keps/sig-cli/2189-kubectl-default-container/README.md

```diff
@@ -0,0 +1,168 @@
+# KEP-2189: default container behavior
+
+## Table of Contents
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+
+<!-- /toc -->
+
+## Summary
+
+Many Kubectl commands make requests to specific containers, `kubectl logs` and `kubectl exec` are most commonly
+used.
+
+All commands that need to specify container name:
+
+- kubectl attach - Attach to a running container
```

Comment:
There are currently 5 commands that allow specifying `--container` flag:
- `kubectl attach`
- `kubectl cp`
- `kubectl debug`
- `kubectl exec`
- `kubectl logs`
I'd like to see an explicit list of commands which will consume this annotation.